services/core-api/app/api/document_generation/namespace.py

```diff
@@ -0,0 +1,7 @@
+from flask_restplus import Namespace
+from app.api.document_generation.resources.now_document import NoticeOfWorkDocumentResource, NoticeOfWorkDocumentGenerationResource
+
+api = Namespace('document-generation', description='Generate documents for business objects.')
+
+api.add_resource(NoticeOfWorkDocumentResource, '/now')
+api.add_resource(NoticeOfWorkDocumentGenerationResource, '/now/<string:document_type_code>')
```

Comment:
Sorry i didn't notice this on the first time. I think we want generate documents to be an action on that document type. 

`/api/now-applications/application-document-types/<code>/generate`

or something like that. Let me know if that's not trivial.

Reply:
We will tackle this when the templates go into the db.